js/signup.js

```diff
@@ -0,0 +1,99 @@
+const emailErrorMessage = document.querySelector(".email-error-message");
+const emailInput = document.querySelector(".email-input");
+
+emailInput.addEventListener("blur", function () {
+  const EMAIL_REGEX = /[a-z0-9]+@[a-z]+\.[a-z]{2,3}/;
+
+  if (emailInput.value === "") {
+    emailErrorMessage.textContent = "이메일을 입력해 주세요.";
+    emailInput.classList.add("error-border");
+    return;
+  }
+
+  if (emailInput.value.match(EMAIL_REGEX) === null) {
+    emailErrorMessage.textContent = "올바른 이메일 주소가 아닙니다.";
+    emailInput.classList.add("error-border");
+    return;
+  }
+
+  if (emailInput.value === "test@codeit.com") {
+    emailErrorMessage.textContent = "이미 사용 중인 이메일입니다.";
+    emailInput.classList.add("error-border");
+    return;
+  }
+
+  emailErrorMessage.textContent = "";
+  emailInput.classList.remove("error-border");
+});
+
+const passwordErrorMessage = document.querySelectorAll(
+  ".password-error-message"
+);
+const passwordInput = document.querySelectorAll(".password-input");
```

Comment:
querySelectorAll 메소드를 사용하는 경우에는 해당 selector에 해당되는 DOM을 모두 가져온다는 의미이기 때문에 변수명이 복수형으로 작성되는게 좋을 것 같아요!
```suggestion
const passwordErrorMessages = document.querySelectorAll(
  ".password-error-message"
);
const passwordInputs = document.querySelectorAll(".password-input");
```